### PR TITLE
Merge radar_tten support for the tenth time.

### PR DESCRIPTION
**Pull request by SamuelTrahanNOAA**
Temperature tendencies in microphysics scheme replaced with radar-derived values for the first N minutes of the model run.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -8,5 +8,5 @@
   branch = main
 [submodule "ccpp/physics"]
   path = ccpp/physics
-  url = https://github.com/NOAA-GSL/ccpp-physics
-  branch = gsl/develop
+  url = https://github.com/SamuelTrahanNOAA/ccpp-physics
+  branch = feature/gsl-develop-20211203-radar-tten
```
Comment (hu5970) [inline .gitmodules:12]: This points to your branch. Is this intended? If Yes, you will change this later, right?

Comment (SamuelTrahanNOAA) [reply .gitmodules:12]: Yes. You have to do this until a merge so people can clone your branch from the ufs-weather-model level.

**Review by hu5970 — APPROVED** [review]

**Comment by climbfuji** [comment]
Note that we should update gsl/develop from the authoritative repositories first before we merge this PR. This is so that we have a version/hash that is 100% identical with the authoritative repository as of beginning of this week.

**Comment by SamuelTrahanNOAA** [comment]
> Note that we should update gsl/develop from the authoritative repositories first before we merge this PR. This is so that we have a version/hash that is 100% identical with the authoritative repository as of beginning of this week.

Yes. Or perhaps I should do a new PR to the community repositories instead?

Either way, someone will have to retest this.

**Comment by hu5970** [comment]
Sam, I can test it again when you have a new PR.

**Comment by hu5970** [comment]
@climbfuji 
I can see you merged gsl/develp branch to community develop branch on December 13, 2021 with hash:
56f3483dacf9bb11597dde731a726cf643049b6d

The gsl/develop branch is still on November 24, 2021 with hash:
0409aa0e59c0096ade9a2ae85cdbc42b1cf75de3

Those two are identical, right? 
I can see the version is identical but why did you say the hash is identical?

Thanks,
Ming

**Comment by climbfuji** [comment]
> @climbfuji I can see you merged gsl/develp branch to community develop branch on December 13, 2021 with hash: 56f3483dacf9bb11597dde731a726cf643049b6d
> 
> The gsl/develop branch is still on November 24, 2021 with hash: 0409aa0e59c0096ade9a2ae85cdbc42b1cf75de3
> 
> Those two are identical, right? I can see the version is identical but why did you say the hash is identical?
> 
> Thanks, Ming

They are not. The Dec 13 hash has changes from the authoritative repositories that have not been merged into gsl/develop yet, in particular the entire revamp of the GSL/RAP regression tests.

**Comment by SamuelTrahanNOAA** [comment]
Closing this so I can do a PR to the community repository.

**Pull request closed without merging**